legacy/Modules/AlphaModule.cs

```diff
@@ -1582,44 +1582,6 @@ public async Task GetRatios()
         // redo insult command
 
         // redo status command, to be compatible with OriUser.
-        [Group("status"), Name("Status"), Alias("st")]
```

Comment:
While I appreciate the help here, I was only referring to multiple classes in a single file. Nested classes are fine as is, since Discord.Net requires that command groups are specified within a base module. Sorry if I caused any confusion. :(

Reply:
Oh. Sorry, my bad. Understood now. Let me fix it.